docs/bigquery-usage.rst

```diff
@@ -25,6 +25,7 @@ Authentication / Configuration
      >>> from gcloud import bigquery
      >>> client = bigquery.Client()
 
+
```

Comment:
They don't affect generated output:  they just delineate sections visually (a bit like PEP8 for top-level module `def` and `class` statements.